package.json

```diff
@@ -52,6 +52,7 @@
     "redux-persist": "4.10.2",
     "redux-persist-transform-filter": "0.0.18",
     "reselect": "4.0.0",
+    "semver": "^6.3.0",
```

Comment:
I know this is how it's merged into master but should we use fix version instead?

Reply:
Good point and good catch, thanks! I had forgotten we do it like that, I'll update.